hack/rbacgenerator/raw.js

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
No idea why it's called that. It's `package.json` in reality. Will rename. Thanks.